client/src/store/modules/scheduledEvent/normalizer.js

```diff
@@ -22,3 +27,18 @@ export const eventPaginationMapper = Pagination => ({
 export const eventEmailMapper = Email => ({
     name: Email.email
 });
+
+export const eventApiMapper = Event => ({
+    id: Event.id,
+    invitee_name: Event.name,
+    invitee_email: Event.email,
+    invitee_question: Event.question,
+    status: Event.status,
+    start_date: moment(Event.startDate)
+        .tz(Event.timezone)
+        .utc()
+        .format('YYYY-MM-DD HH:mm:ss'),
+    location: Event.location,
+    timezone: Event.timezone,
+    event_type: eventTypeMapper(Event.eventType)
```

Comment:
I'm not sure if we have to process data on the client side

Reply:
We need push data from client in UTC format :)